src/widgets/courses/ui/courses.module.scss

```diff
@@ -0,0 +1,42 @@
+.all-courses {
+  @include media-laptop {
+    padding: 40px;
+  }
+
+  @include media-tablet {
+    padding: 0;
+  }
+}
+
+.courses-content {
+  padding-top: 40px;
+
+  @include media-laptop {
+    flex-wrap: wrap;
+    gap: 24px;
+    width: 100%;
+    padding: 8px 6px;
+  }
+
+  @include media-tablet {
+    padding: 0 16px;
+  }
+}
+
+.course-list {
+  display: grid;
+  grid-auto-flow: row;
```

Comment:
redundant because we explicitly define the rows below with `grid-template`

```suggestion

```

Reply:
Done